lib/build-group-id.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable camelcase */
+
+const { unboxKey, DeriveSecret, CloakedMsgId } = require('envelope-js')
+const LABELS = require('envelope-spec/derive_secret/constants.json')
+const { keySchemes } = require('private-group-spec')
+
+const bfe = require('ssb-bfe')
+
+module.exports = function buildGroupId ({ groupInitMsg, readKey, msgKey, groupKey }) {
+  const msgId = bfe.encode(groupInitMsg.key)
+
+  if (!readKey) {
+    if (groupInitMsg.value.meta && groupInitMsg.value.meta.unbox) {
```

Comment:
This function was copy pasted from ssb-tribes, but it doesn't work. ssb-db2 doesn't have `msg.value.meta.unbox`.